.github/workflows/build.yml

```diff
@@ -0,0 +1,26 @@
+name: ble.sh build
+on:
+  push:
+    branches:
+      - master
+
+jobs:
+  builds:
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout
+        uses: actions/checkout@v3
+        with:
+          submodules: recursive
+      - name: Package
+        run: |
+          mkdir ble.sh
+          make install INSDIR="$(pwd)/ble.sh"
+          tar -cJf ble.tar.xz ble.sh
```

Comment:
```suggestion
          tar -cJf ble.tar.xz "ble-nightly+$(git rev-parse --short HEAD).tar.xz"
```

Reply:
Yes, I notice that but I want to have a PR so we can discuss about details like this

Reply:
I see. Thanks for opening the PR!